instagram/src/components/CommentSection/CommentSection.js

```diff
@@ -0,0 +1,41 @@
+import React from "react";
+import Comment from "../Comment/Comment";
+import PropTypes from "prop-types";
+import moment from "moment";
+
+const CommentSection = ({
+  comments,
```

Comment:
I'd recommend refactoring this component to a stateful class component so you can hold each instance of newComment locally instead of in one place in App.js, like I mentioned above